packages/cloud-cognitive/src/components/Datagrid/Datagrid/DraggableElement.js

```diff
@@ -12,6 +12,7 @@ import { Draggable16 } from '@carbon/icons-react';
 import { useDrag, useDrop } from 'react-dnd';
 import cx from 'classnames';
 import { pkg } from '../../../settings';
+// import { TableSelectRow } from 'carbon-components-react';
```

Comment:
if the  `TableSelectRow` no longer used this line should be deleted.

(the less comments and less commented out code, the easier it is to parse the module)